pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "manim"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
Shouldn't this be 0.3.0? We didn't do a mid-month release.

Reply:
If we're using semantic versioning, then points 4, 6 and 7 are [useful references](https://semver.org/#spec-item-4). I think this release adds a few new backwards compatible features and would justify bumping the minor instead. Perhaps future month releases will only involve bug fixes/patches, but I foresee **many** minor bumps as we add new features each month to the core (I'm calling it here, 0.11.2 -> 1.0.0)

Reply:
As long as our major version is still 0, that doesn't really apply though necessarily. That being said, I'd be fine with bumping to v0.3.0 (although I was thinking of moving to v0.3.0 only with the switch to the webgl renderer).

Reply:
```suggestion
version = "0.3.0"
```

Reply:
From the discussion on Discord it seems that people are either neutral or leaning towards `v0.3.0` -- so let's adhere to semantic versioning, respect that we've introduced new, backwards-compatible features, and let's go with `v0.3.0`. I'm pushing another commit.